<launch>
  <group ns="renov_up_level">
    <!-- mat document reading parameters -->
    <param name="mat_data_path" value="/home/zy/catkin_ws/src/paintingrobot/painting_robot_demo/data/data.mat"/>
    <param name="coverage_planner_path" value="/home/zy/catkin_ws/src/paintingrobot/painting_robot_demo/scripts"/>
    
    <!-- <param name="mat_data_path" value="/data/ros/renov_robot_ws/src/painting_robot_demo/data/data.mat"/>
    <param name="coverage_planner_path" value="/data/ros/renov_robot_ws/src/painting_robot_demo/scripts"/> -->
  
    <param name="mat_parameterx" value="0.430725381079" />
    <param name="mat_parametery" value="-0.00033063639818" />
    <param name="mat_parameterz" value="0.028625" />
    <param name="mat_interval" value="0.10" />


    <!-- aubo parameters -->
    <param name="aubo_ip" value='192.168.1.11'/>
    <param name="aubo_start_point" value='(0.0,0.7156,2.7524,0.47,-1.487,1.57)'/> 
    <param name="aubo_end_point" value='(0.0,0.7156,2.7524,0.47,-1.487,0)' /> 

    <param name="joint_maxacc_tuple" value='(2.5, 2.5, 2.5, 2.5, 2.5, 2.5)'/> 
    <param name="joint_maxvelc_tuple" value='(1.5, 1.5, 1.5, 1.5, 1.5, 1.5)'/> 
    <param name="ee_maxacc" value='0.2'/> 
    <param name="ee_maxvelc" value='0.2'/>
    <param name="blend_radius" value='0.05'/> 

    <!-- FSM parameters -->
    <param name="mobile_platform_tracking_over_flag" value="0"/>
    <param name="rodmechanism_holding_over_flag" value="0"/>
    <param name="climbingmechanism_climbing_over_flag" value="0"/>
    <param name="manipulator_renovation_over_flag" value="0" />
    <param name="one_mobilebase_operation_over_flag" value="0" />
    <param name="jackup_mechanism_homing_over_flag" value="0" />

    <param name="last_motion_phase_over_flag" value="0" />
    <param name="current_motion_phase_start_flag" value="0" />
    <param name="current_motion_phase_over_flag" value="0" />

    <node pkg="painting_robot_demo" type="coverage_painting_demo1.py" name="coverage_painting_simulation" output="screen" >
    </node>

  <!-- other parameters -->
  <!-- <param name="mobile_tracking_stop_flag" value="0" />
  <param name="painting_opreating_over" value="0"/> -->

  <!-- set /global_example parameter -->
  <!-- <include file="$(find painting_robot_demo)/launch/search_usb_port.launch">
  </include>
  <include file="$(find painting_robot_demo)/launch/plc_node_zy.launch">
  </include> -->
  <!-- <include file="$(find painting_robot_demo)/launch/mobile_platform_driver.launch">
  </include> -->
  <!-- <include file="$(find painting_robot_demo)/launch/3dof_mechanism_startup_zy.launch">
  </include>
  <include file="$(find painting_robot_demo)/launch/aubo_startup_zy.launch">
  </include>  -->

  </group>
</launch>

